tools/macos_buildenv.sh

```diff
@@ -90,26 +87,18 @@ case "$1" in
             rm "${SDKROOT}.tar.xz"
         fi
 
-        Qt5_DIR="$(find "${BUILDENV_PATH}" -type d -path "*/cmake/Qt5")"
-        [ -z "${Qt5_DIR}" ] && echo "Failed to locate Qt5_DIR!" >&2
-        QT_QPA_PLATFORM_PLUGIN_PATH="$(find "${BUILDENV_PATH}" -type d -path "*/plugins")"
-        [ -z "${QT_QPA_PLATFORM_PLUGIN_PATH}" ] && echo "Failed to locate QT_QPA_PLATFORM_PLUGIN_PATH" >&2
-        export CC="/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
-        export CXX="/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++"
-        export PATH="${BUILDENV_PATH}/bin:${PATH}"
-        export CMAKE_PREFIX_PATH="${BUILDENV_PATH}"
-        export Qt5_DIR
-        export QT_QPA_PLATFORM_PLUGIN_PATH
+        export VCPKG_ROOT="${BUILDENV_PATH}"
+        export VCPKG_OVERLAY_TRIPLETS="${BUILDENV_PATH}/overlay/triplets"
+        export VCPKG_DEFAULT_TRIPLET=x64-osx
+        export X_VCPKG_APPLOCAL_DEPS_INSTALL=ON
+        export CMAKE_GENERATOR=Ninja
 
         echo_exported_variables() {
-            echo "CC=${CC}"
-            echo "CXX=${CXX}"
-            echo "SDKROOT=${SDKROOT}"
```

Comment:
This was lost... I'm confused how the build isn't unusably slow...???? But if it works, yay? I guess?